lib/runner.js

```diff
@@ -46,6 +46,10 @@ var Runner = function(config) {
     flow.timeout(1000, 'waiting for debugger to attach');
   }
 
+  if (config.debuggerServerPort) {
+    process.debuggerServerPort = config.debuggerServerPort;
```

Comment:
`process.debugPort` is actually used by node debugger to get the port to create the debugger at, so that can't be changed. 
For debuggerServerPort though, I created `browser.debuggerServerPort_`.